src/settings.cpp

```diff
@@ -934,20 +934,20 @@ void CServerSettings::ReadSettingsFromXML ( const QDomDocument& IniXMLDocument,
                                     static_cast<int> ( AT_DEFAULT ),
                                     static_cast<int> ( AT_CUSTOM ),
                                     iValue ) )
-        {
-            directoryType = static_cast<EDirectoryType> ( iValue );
-        }
-        //### TODO: END ###//
+            {
```

Comment:
So this is the one I think is questionable...  I think.

Reply:
Probably. I'd just add a `{` after the else - also for clarity.

In my own code, I only omit `{` if it's possible to write the statement in the same line as the else but still consider it "lazy styling":
```c
if (something()) call_true();
```

Reply:
The blocks here are
- 911-915 (open/close align)
- 916-960 (open/close align)
    - 918-925 (`//### TODO: BEGIN ###//` and `//### TODO: END ###//` do _not_ align.  This is because the END is in the middle of an `else ... if`)
    - 927-940 (note line 942 should be split as 924 and 931 are, causing the same issue; open/close align)
    - 942-950 (open/close align)
    - 952-959 (open/close align)

Reply:
So I don't think this is a problem now, resolving.

Reply:
It still looks confusing. Did you open an issue on this?